### PR TITLE
Test/reducer action

### PR DESCRIPTION
**Pull request by Kyl23**
## Description
- test SET_POST_FORM
- test TOGGLE_STATIS_DATA （include alertWord)
- OVERWRITE_POST_FORM （didn't write test cuz api haven't connect yet)
## Github issue
https://github.com/JiaAnTW/NCKU_Trans/issues/46
https://github.com/JiaAnTW/NCKU_Trans/issues/100
## Scope
- /
- /post
- /admin/*
- component
- Global layout
    - SideBar  

## Screenshot
image

### CODE REVIEW

File: resources/assets/js/_tests_/model/reducer/initPost.js [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+        case 'onChange_other':
+            return (state, index, value) => {
+                const stateNext = state;
+                const { type, step } = stateNext;
+                stateNext.form[type].pageMap[step / 2][2][index].value = value;
+            };
+        default:
+            return undefined;
+    }
+}
```
Comment (JiaAnTW) [inline resources/assets/js/_tests_/model/reducer/initPost.js:22]: 這個function應該不用重新定義? import原始的就好，之前initCommit之所以需要這樣做是因為我需要IIFE執行完的值

File: resources/assets/js/_tests_/model/reducer/initPost.js [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+export const initComment = (function () {
+    const yearOption = [];
```
Comment (JiaAnTW) [inline resources/assets/js/_tests_/model/reducer/initPost.js:24]: initComment跟initStudy感覺可以分檔?這樣比較好trace code

Comment (Kyl23) [reply resources/assets/js/_tests_/model/reducer/initPost.js]: done

**Comment by Kyl23** [comment]
> > 
> 
> ok

**Review by JiaAnTW — APPROVED** [review]

**Pull request merged**